src/routes/safe/store/actions/fetchTransactions.js

```diff
@@ -147,29 +147,54 @@ export const buildTransactionFrom = async (
     modifySettingsTx,
     customTx,
     cancellationTx,
+    creationTx: tx.creationTx,
   })
 }
 
+const addMockSafeCreationTx = (safeAddress) => [{
```

Comment:
Maybe we should add it at the point when safe is loaded/created, not on every TXs fetch. Because this tx is more of "add once - forget about it" thing. I don't think we should complicate `fetchTransactions` method with it because it polls every 5s and does a lot of things already